mule.yaml

```diff
@@ -29,6 +34,11 @@ stages:
     version: '{{ docker.Version.linux-arm64.version }}'
     workDir: /go/src/github.com/algorand/go-algorand
     target: fulltest ci-build
+  - task: docker.Make
```

Comment:
I think having these as separate mule jobs would be more convenient for us showing logs in our ci server

Reply:
Actually it looks like the ci-integration job will only work on the host/container that build the binaries, since it looks in the gopath. I think that's ok for now

Reply:
oh nvm, I see that it is actually using the node_pkg dir

Reply:
OK, split integration tests into separate jobs.